sample_code/s3_to_redshift.py

```diff
@@ -0,0 +1,48 @@
+import os
+import logging
+from parsons import Redshift, S3, utilities
+
+# Redshift setup - this assumes a Civis Platform parameter called "REDSHIFT"
+
+set_env_var(os.environ['REDSHIFT_PORT'])
+set_env_var(os.environ['REDSHIFT_DB'])
+set_env_var(os.environ['REDSHIFT_HOST'])
+set_env_var(os.environ['REDSHIFT_CREDENTIAL_USERNAME'])
+set_env_var(os.environ['REDSHIFT_CREDENTIAL_PASSWORD'])
+rs = Redshift()
+
+# AWS setup - this assumes a Civis Platform parameter called "AWS"
+
+set_env_var('S3_TEMP_BUCKET', 'parsons-tmc')
+set_env_var('AWS_ACCESS_KEY_ID', os.environ['AWS_ACCESS_KEY_ID'])
+set_env_var('AWS_SECRET_ACCESS_KEY', os.environ['AWS_SECRET_ACCESS_KEY'])
+s3 = S3()
+
+# Logging
+
+logger = logging.getLogger(__name__)
+_handler = logging.StreamHandler()
+_formatter = logging.Formatter('%(levelname)s %(message)s')
+_handler.setFormatter(_formatter)
+logger.addHandler(_handler)
+logger.setLevel('INFO')
+
+
+bucket = os.environ['BUCKET']
+schema = os.environ['SCHEMA']
+
+keys = s3.list_keys(bucket)
+files = keys.keys()
+
+if len(keys) == 0:
+    logger.info("No files to sync today!")
+else:
+    logger.info(f"Pulling {str(len(files))} files down from s3...")
+    for x in files:
```

Comment:
Should we also include the file closer utility, in case it's a lot of files? Or is that too complicated for this sample code?